drafts/zip-str4d-sapling-anchor.rst

```diff
@@ -0,0 +1,110 @@
+::
+
+  ZIP: XXX
+  Title: Sapling Anchor Deduplication within Transactions
+  Owners: Jack Grigg <jack@z.cash>
+  Status: Draft
+  Category: Consensus
+  Created: 2019-03-27
+  License: MIT
+
+
+Terminology
+===========
+
+The key words "MUST" and "MAY" in this document are to be interpreted as described in
+RFC 2119. [#RFC2119]_
+
+The term "network upgrade" in this document is to be interpreted as described in ZIP 200
+[#zip-0200]_.
+
+The term "Sapling" in this document is to be interpreted as described in ZIP 205
+[#zip-0205]_.
+
+Abstract
+========
+
+This proposal defines a modification to the transaction format whereby a single Sapling
+anchor is used for all Sapling spends. This change removes a potential implementation
+fingerprint, and reduces the size of Sapling transactions within the block chain.
```

Comment:
Remove comma between "fingerprint" and "and." I believe "blockchain" is a legitimate word.

Reply:
The spec and ZIPs use "block chain" by convention.